src/multibody/joint/joint-free-flyer.hpp

```diff
@@ -192,7 +192,7 @@ namespace pinocchio
     : M(1)
     , U(U_t::Zero())
     , Dinv(D_t::Zero())
-    , UDinv(UD_t::Zero())
+    , UDinv(UD_t::Identity())
```

Comment:
For consistency, shouldn't it be UDinv = U * Dinv ?

Reply:
When you do the math, this quantity is always equal to Identity ;)

Reply:
I understand it and I do agree with this change. What I mean is, shouldn't you change U and Dinv too, so to be consistent?

Reply:
U and UDInv depends on the Inertia I input ;)

Reply:
Oh, I see. Well, as they say, sorry for the noise